scripts/seed-data

```diff
@@ -18,4 +18,8 @@ cloud_init:
     - file:/var/lib/rancher/conf/$(basename ${CLOUD_DATA})
 EOF
 
+for f in ${FILES[@]}; do
+  cp -a ${f} ${BASE_DIR}/lib/rancher/conf/
```

Comment:
No worries, @ibuildthecloud is working on dropping cloudconfig files into a config.d dir... which it might well be `/lib/rancher/conf/config.d` :)